tests/test_models/test_nn/tft_native/test_tft_native.py

```diff
@@ -188,31 +188,37 @@ def test_tft_make_samples(
             df[[feature]].iloc[:1],
             first_sample["static_reals"][feature],
         )
+        assert first_sample["static_reals"][feature].base is not None
```

Comment:
Don't we want to add such tests for other models too? It seems like a good practice to follow for future models.

Reply:
Good idea!